geocube/geo_utils/geobox.py

```diff
@@ -37,19 +39,18 @@ def geobox_from_rio(xds):
 
     Returns
     -------
-    :obj:`datacube.utils.geometry.GeoBox`
+    :obj:`odc.geo.geobox.GeoBox`
 
     """
     height, width = xds.rio.shape
     try:
         transform = xds.rio.transform()
     except AttributeError:
         transform = xds[xds.rio.vars[0]].rio.transform()
-    return geometry.GeoBox(
-        width=width,
-        height=height,
+    return GeoBox(
+        shape=XY(x=width, y=height),
```

Comment:
there is `wh_(width, height)` helper that produces shape class, I recommend using that instead of  XY class.
high level docs are still missing, but there is migration guide here:
https://odc-geo.readthedocs.io/en/latest/migration.html